packages/api/src/transfer/transfer.entity.ts

```diff
@@ -53,9 +53,9 @@ export class Transfer extends BaseEntity {
   @Column({ type: "varchar", length: 128, nullable: true })
   public readonly amount?: string;
 
-  @ManyToOne(() => Token)
+  @ManyToOne(() => Token, { createForeignKeyConstraints: false })
```

Comment:
By default ManyToOne creates a foreign key, but in worker schema this entity doesn't have it:
1. it creates a discrepancy between e2e DB and DB created by worker.
2. requires creating some additional entities in e2e tests even if they are not used.

To solve this issue createForeignKeyConstraints relation option is used so DB created by API for e2e and DB created by worker match.